Source/qol/monhealthbar.cpp

```diff
@@ -135,7 +135,7 @@ void DrawMonsterHealthBar(const Surface &out)
 	DrawString(out, monster.mName, { position + Displacement { -1, 1 }, { width, height } }, style | UiFlags::ColorBlack);
 	if (monster._uniqtype != 0)
 		style |= UiFlags::ColorWhitegold;
-	else if (monster.leader != 0)
+	else if (monster.leader != Monster::NoLeader)
 		style |= UiFlags::ColorBlue;
 	else
 		style |= UiFlags::ColorWhite;

```

Comment:
Previously monsters which were part of a group would keep the blue text on the health bar when their leader died. With the last commit they fall back to white. I didn't think this was worth trying to preserve, I'd rather not keep a reference to a dead monster around purely for an optional QoL feature.

Reply:
They would still have increased stats, so would be best if there was still a way to indicate this.

Reply:
Can add a flag to the Monster struct that gets set if they were spawned as part of a group, or maybe monhealthbar should be comparing stats to what is expected for that type of monster?

Reply:
Was having a think about this and comparing stats *should* work for monhealthbar, but I need to read through the code to make sure every group does in fact have different int compared to when they're naturally spawned (there's a couple of unique bosses with 0 and normal spawns have 0, but they might not overlap). I'll revise this PR to leave the dangling leader reference with a comment about the use when I get time this evening.